adm/src/main.rs

```diff
@@ -107,5 +112,47 @@ fn parse_args<'a>() -> ArgMatches<'a> {
               settings transaction family other than sawtooth_settings)"))
         (@arg verbose: -v... "increase the logging level.")
     );
+
+    #[cfg(feature = "client-cli")]
+    let app = app.subcommand(
+        SubCommand::with_name("batch")
+            .about(
+                "display batch information and submit batches to the
+                    validator via the REST API",
+            )
+            .setting(AppSettings::SubcommandRequiredElseHelp)
+            .subcommand(
+                SubCommand::with_name("list")
+                    .about(
+                        "display all information about all committed Batches for\n\
+                            the specified validator, including the Batch id, public\n\
+                            keys of all signers, and number of transactions in each Batch",
+                    )
+                    .arg(Arg::with_name("url").long("url").takes_value(true).help(
+                        "identify the URL of the validator's\n\
+                            REST API (default:\n\
+                            http://localhost:8008)",
+                    ))
+                    .arg(
+                        Arg::with_name("username")
+                            .long("user")
+                            .short("u")
+                            .takes_value(true)
+                            .value_name("USERNAME[:PASSWORD]")
+                            .help(
+                                "specify the user to authorize\n\
+                                request",
+                            ),
+                    )
+                    .arg(
+                        Arg::with_name("format")
+                            .long("format")
+                            .short("F")
+                            .takes_value(true)
+                            .value_name("csv,json,yaml,default")
```

Comment:
This should be `.possible_values(...)` https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.possible_values

Reply:
Fixed